roles/taca/templates/sthlm_fastq_delivery.yml.j2

```diff
@@ -6,7 +6,7 @@ deliver:
     analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
     datapath: <ROOTPATH>/DATA/<PROJECTID>
     stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
-    deliverypath: /pica/sw/links/proj/<UPPNEXID>/INBOX/<PROJECTID>
+    deliverypath: "{{ item.proj_path }}/<UPPNEXID>/INBOX/<PROJECTID>"
```

Comment:
Don't use quotes (`" "`) if you don't want them in the generated template. It will not be parsed by Jinja.